src/navbar/menu.styles.tsx

```diff
@@ -0,0 +1,162 @@
+import styled from "styled-components";
+import { Color } from "../color";
+import { MediaQuery } from "../media";
+import { Text, TextStyleHelper } from "../text";
+import { DesignToken } from "src/design-token";
+// =============================================================================
+// STYLE INTERFACE, transient props are denoted with $
+// See more https://styled-components.com/docs/api#transient-props
+// =============================================================================
+interface StyleProps {
+    $selected: boolean;
+}
+
+// =============================================================================
+// WRAPPER
+// =============================================================================
+
+export const Wrapper = styled.ul`
+    display: flex;
+    list-style: none;
+    position: relative;
+
+    ${MediaQuery.MaxWidth.tablet} {
+        display: none;
+    }
+`;
+
+export const MobileWrapper = styled.ul`
+    display: none;
+    list-style: none;
+
+    ${MediaQuery.MaxWidth.tablet} {
+        border-left: 0.25rem solid ${Color.Primary};
+        display: flex;
+        flex-direction: column;
+        padding-bottom: 1rem;
+    }
+`;
+
+// =============================================================================
+// LINK ITEMS
+// =============================================================================
+
+export const Link = styled(Text.Hyperlink.Small)<StyleProps>`
+    ${TextStyleHelper.getTextStyle("H6", 400)};
+    width: 100%;
+    display: flex;
+    position: relative;
+    align-items: left;
+    text-align: left;
+    color: ${Color.Neutral[1]};
+
+    padding-left: 1rem;
+    padding-right: 1rem;
+    padding-top: 1px;
+    padding-bottom: 1px;
```

Comment:
can simplify to a single `padding` property. also is the 1px needed or can it just be 0?